### ISSUE TITLE
[Checkbox] When using React-md Checkbox the id is not being set

### ISSUE THREAD

**Issue by jbetancur** [jbetancur/react-data-table-component#3]
* When using React-md Checkbox the id is not being set

```
Failed prop type: The prop `id` is required to make `Checkbox` accessible for users of assistive technologies such as screen readers.
```

**Comment by jbetancur** [comment]
This is a react-md issue

**Comment by Angelk90** [comment]
Hi @jbetancur , how did you solve?

**Comment by jbetancur** [comment]
Unfortunately, I was not able to as the issue likely internal to react-md. Not really an issue on their end per say, but doesn’t play well with this lib in terms of that warning. It’s safe to ignore it though. Note that material ui checkbox does not exhibit this but To be honest I haven’t had the time to look into this further

**Comment by Angelk90** [comment]
@jbetancur : As I understand it, we would like each checkbox to have a unique key.

**Comment by jbetancur** [comment]
Yes, but if I pass a unique Id to the checkbox with DataTable the check boxes break.
